### ISSUE TITLE
Search Name (see_results function) returns timeout error

### ISSUE THREAD

**Issue by MichaelArenander** [IBM-Bluemix/nodejs-cloudantdb-crud-example#3]
Hi, after forking this project in Bluemix, changing host and then adding names to the database, I get this error alert when trying to search for my added names using the project's Search Name tab:

```
Error is : {
  "readyState": 0,
  "status": 0,
  "statusText": "timeout"
}
```

It worked completely fine for me two weeks ago, but somewhere in the beginning of October the see_results function in "public/index.html" didn't seem to work properly by either returning the error alert above, or by not returning anything at all, while always giving the 'process complete' message in the console for every attempt. Also, my Bluemix Log doesn't return any errors or response when executing the see_results function.

I have also tried the master project at [http://nodejscloudantsampleapp.mybluemix.net/](http://nodejscloudantsampleapp.mybluemix.net/) which seems to have stopped working by the same time as my forked project. 

I appreciate the work put into this project, it has helped me a lot with the understanding of basic NoSQL for webpages.

**Comment by rvennam** [comment]
I just deployed a new instance and it seems to be working OK: http://nodejs-cloudantdb-crud-example-rvennam-836.mybluemix.net/ Can you retry by creating a new app?

**Comment by MichaelArenander** [comment]
Yes, I see now that this wasn't the same project as the one I forked, although similar. I got mine from [NodeJSCloudantSampleApp](https://hub.jazz.net/project/neerajaganesan/NodeJSCloudantSampleApp/overview) which seems to have its own github [project link](https://github.com/nganesan9/NodeJSCloudantSampleApp-master) where this issue belongs. Since the part which i have an issue with isn't included in this crude-example version, I can as well call it closed. I appreciate the quick response and may look into this project instead if my issue persists.
